### ISSUE TITLE
Pytorch version Compatibility

### ISSUE THREAD

**Issue by jmlee99o** [Ldhlwh/DFMGAN#41]
I use PyTorch 2.3.1+cu118, 'conv2d_gradfix' and 'grid_sample_gradfix' operations are not supported, instead using basic 'torch.nn.functional.conv2d' and 'torch.nn.functional.grid_sample' operations. Also, RuntimeError has occurred that no function has been implemented to calculate the gradient for aten:grid_sampler_2d_backward.

Which PyTorch, Cuda version should I use to avoid errors and use that model well?

**Comment by Ldhlwh** [comment]
Refer to https://github.com/Ldhlwh/DFMGAN#getting-started

**Comment by jmlee99o** [comment]
Thanks
